doc/12-icinga2-api.md

```diff
@@ -1129,13 +1132,13 @@ In addition to these parameters a [filter](12-icinga2-api.md#icinga2-api-filters
 Example for a custom host notification announcing a global maintenance to
 host owners:
 
-```
-$ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
+```bash
+curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
  -X POST 'https://localhost:5665/v1/actions/send-custom-notification' \
  -d '{ "type": "Host", "author": "icingaadmin", "comment": "System is going down for maintenance", "force": true, "pretty": true }'
 ```
 
-```
+```json
```

Comment:
The following block contains invalid JSON: The result list is missing the closing `]`.

I've stopped reviewing here as this somehow messes up the diff rendering for the remaining file.